frontend/src/slices/auth/actions.ts

```diff
@@ -38,12 +38,26 @@ const signIn = createAsyncThunk<
 });
 
 const getCurrentUser = createAsyncThunk<
-  UserAuthResponseDto,
+  UserAuthResponseDto | null,
   undefined,
   AsyncThunkConfig
 >(`${sliceName}/getCurrentUser`, async (_loginPayload, { extra }) => {
-  const { authApi } = extra;
-  return await authApi.getCurentUser();
+  const { authApi, storage } = extra;
+
+  const token = await storage.get(StorageKey.TOKEN);
+  const hasToken = Boolean(token);
+
+  if (!hasToken) {
+    return null;
+  }
+
+  try {
+    return await authApi.getCurentUser();
```

Comment:
Missed typo before :c
```suggestion
    return await authApi.getCurrentUser();
```

Reply:
fix - [6bb6359](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/94/commits/6bb63592cd3477979337716a326a29c8e06b3f8a)